client/my-sites/site-settings/section-import.jsx

```diff
@@ -69,21 +66,25 @@ export default React.createClass( {
 		}
 
 		return status.map( item => Object.assign( {}, item, { site, siteTitle } ) );
-	},
+	}
 
-	updateFromAPI: function() {
+	updateFromAPI = () => {
 		fetchState( this.props.site.ID );
-	},
+	};
```

Comment:
I have no preferences about that really. My point there is just as @mcsf explained (thanks for that) - it's an assignment and not a method; and the same applies for class properties, so using semicolons there makes sense. 

But it is 2 vs 1 here, so I removed them in https://github.com/Automattic/wp-calypso/pull/13182/commits/ec3a280c8f6199cd8c5711c2fbf1ad8f15ee80dd. 😉